ci_framework/playbooks/02-infra.yml

```diff
@@ -25,6 +25,10 @@
       when:
         - cifmw_pkg_build_list is defined
         - cifmw_pkg_build_list | length > 0
+    - name: Make sure we have all the dependencies needed by install_yamls
```

Comment:
Hi Pablo,

Thank you for your suggestion.

In my new job, we utilize similar playbooks as the ones used in the cifmw-crc-podified-edpm-deployment job. The only difference is that there is a toggle between 07-deploy-edpm-with-bmaas.yml and 06-deploy-edpm.yml, based on the edpm_compute_baremetal job variable in the deploy-edpm.yml file.

I noticed that @raukadah has already proposed another PR (https://github.com/openstack-k8s-operators/ci-framework/pull/201/files#diff-c857c8304c34f1263cc0b9b81ae71ce6f52a936cb7e540627995c7a3d19e20d4) to run steps from download_tools.yaml. Since there is already a PR in progress, let's wait and see if @raukadah's PR can be merged first (by tomorrow). Once it does, I can either remove this step from my PR, rebase my PR on his, or incorporate the code from his PR into mine. This way, it won't be an issue whether his PR merges first or mine does.

Please let me know if you have any further suggestions or if you need any additional information.

Reply:
If you are gonna work on top of @raukadah PR it's OK for me with that approach.

Reply:
yep, that's for a hook, because we may NOT need to get the tools - they may be pre-existing and maintained by the user.